tests/integration/test_agentd/test_agentd_enrollment_params.py

```diff
@@ -87,7 +87,7 @@ def clean_log_file():
     try:
         client_file = open(LOG_FILE_PATH, 'w')
         client_file.close()
-    except IOError as exception:
+    except IOError:
```

Comment:
You can simplify all this function by removing the `try/except` block and just doing this:
```python
open(LOG_FILE_PATH, 'w').close()
```

Reply:
https://github.com/wazuh/wazuh-qa/pull/1319/commits/21084af927121694ca723501b2bcc3603c0d1c13